src/components/modals/VerseModal.tsx

```diff
@@ -8,14 +8,22 @@ type Props = {
 }
 
 export const VerseModal = ({ isOpen, handleClose, verseText }: Props) => {
+  const copiedTextMessage = `Thou shalt not steal or attempt to cheat at ThyWordle.com`
+
   return (
     <BaseModal title="Verse" isOpen={isOpen} handleClose={handleClose}>
       <img
         src={openQuote}
         alt=""
         className="absolute left-4 top-8 w-10 md:top-16"
       />
-      <div className="transform overflow-hidden px-4 pt-5 pb-4 text-left align-bottom transition-all sm:my-8 sm:w-full sm:max-w-md sm:p-6 sm:align-middle">
+      <div
+        className="transform overflow-hidden px-4 pt-5 pb-4 text-left align-bottom transition-all sm:my-8 sm:w-full sm:max-w-md sm:p-6 sm:align-middle"
+        onCopy={(e) => {
+          e.preventDefault()
+          navigator.clipboard.writeText(copiedTextMessage)
```

Comment:
It really is this easy to change what is copied to the clipboard. This is really fun to me!

Reply:
very cool! I had no idea this was so easy.